recipes/polars/bld.bat

```diff
@@ -0,0 +1,6 @@
+set "CMAKE_GENERATOR=NMake Makefiles"
+maturin build --no-sdist --release --strip --manylinux off --interpreter=%PYTHON% --rustc-extra-args="-C codegen-units=16 -C lto=thin -C target-cpu=native"
```

Comment:
Why `cpu=native`?

Reply:
I disabled windows builds for now, and I haven't updated the bld.bat to match the build.sh script.

Reply:
Maybe we should drop this file then?

Reply:
No just leave it, we plan to add windows shortly its just hitting some permissions weirdness and we don't want to block

Reply:
Then you should use the same flags as on Linux and remove the `cpu-native`.

Reply:
will do